contracts/src/v0.8/automation/dev/test/BaseTest.t.sol

```diff
@@ -173,21 +173,24 @@ contract BaseTest is Test {
       flatFeeMilliCents: DEFAULT_FLAT_FEE_MILLI_CENTS, // 2 cents
       priceFeed: address(USDTOKEN_USD_FEED),
       fallbackPrice: 100_000_000, // $1
-      minSpend: 1000000000000000000 // 1 USD
+      minSpend: 1000000000000000000, // 1 USD
+      decimals: 18
```

Comment:
Can we make this guy 6 or 8 or something small? To kind of match our expectations for some USD tokens in the wild?